manifests/varnish-modules.pp

```diff
@@ -0,0 +1,34 @@
+# Class: yumrepos::varnish-modules
+#
+# Installs the varnish-modules yum repository.
+#
+class yumrepos::varnish-modules (
```

Comment:
I think we need to keep the varnish version in the class name (and all the variables in `params.pp`). How about `varnish-4.1.5-varnish-modules`? Maybe with underscores though to match existing variables in other classes?